src/styles/variables.scss

```
@@ -0,0 +1,24 @@
+$calendar-cell: 100px;
+$shift: 1px;
+$calendar-width: $calendar-cell * 7 + $shift * 6;
+$weekday: (
+  'mon': 0,
+  'tue': (
+    $calendar-cell + $shift,
+  ),
+  'wed': (
+    $calendar-cell * 2 + $shift * 2,
+  ),
+  'thu': (
+    $calendar-cell * 3 + $shift * 3,
+  ),
+  'fri': (
+    $calendar-cell * 4 + $shift * 4,
+  ),
+  'sat': (
+    $calendar-cell * 5 + $shift * 5,
+  ),
+  'sun': (
+    $calendar-cell * 6 + $shift * 6,
+  ),
```

These calculations must be done inside a loop so as not to duplicate the code
```suggestion
  'mon': 0,
  'tue': 1,
  'wed': 2,
  'thu': 3,
  'fri': 4,
  'sat': 5,
  'sun': 6,
```